main.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"encoding/base64"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/ONSdigital/rm-survey-service/models"
+	"github.com/cloudfoundry-community/go-cfenv"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+)
+
+const (
+	realm       = "sdc"
+	serviceName = "surveysvc"
+	timeFormat  = "2006-01-02T15:04:05Z0700"
+)
+
+var logger *zap.Logger
```

Comment:
this should be removed as it's leftover from when the echo framework was used - theres a logger that digital publishing use which we can use https://github.com/ONSdigital/go-ns/tree/master/log

Reply:
Or just use the standard logger until you want something fancy